analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd

```diff
@@ -0,0 +1,386 @@
+                            "Deceased, Progressive", "Recurrence, Second Malignancy", "Progressive, Recurrence", 
+                            "Diagnosis, Second Malignancy", "Deceased, Progressive, Recurrence", "Diagnosis, Progressive, Recurrence",
+                            "Deceased, Recurrence", "Deceased, Diagnosis, Progressive, Recurrence", "Deceased, Diagnosis, Recurrence",
+                            "Deceased, Diagnosis, Progressive", "Diagnosis, Unavailable")) %>%
+  mutate(experimental_strategy = gsub("RNA-Seq;","", experimental_strategy)) %>%
+  write_tsv(file.path(results_dir, "genomic_assays_matched_time_points.tsv"))
+
+genomic_assays_matched %>%
+  group_by(Kids_First_Participant_ID, descriptors, experimental_strategy) %>%
+  tally() %>%
+  arrange(desc(n)) %>%
+  regulartable() %>%
+  fontsize(size = 12, part = "all")
+
+matched_samples_genomic <- length(unique(genomic_assays_matched$Kids_First_Participant_ID))
+```
+## How many samples have genomic assays and matched longitudinal information?
+There are `r matched_samples_genomic` patient cases with matched longitudinal samples for genomic assays.
+
+# Transcriptomic assays
+We will look only at RNA-seq samples and perform the same collapsing of the tumor_descriptor column.
+```{r}
+transcriptomic_assays <- tumor_descriptor_df %>%
+  filter(experimental_strategy_sum %in% c("Both", "RNA")) %>%
+  group_by(Kids_First_Participant_ID) %>%
+  summarize(descriptors = paste(sort(unique(tumor_descriptor)), collapse = ", "),
+            experimental_strategy = unique(experimental_strategy),
+            cancer_group = paste(sort(unique(cancer_group)), collapse = ", "))
+
+kids_pbta_samples_rna <- length(unique(transcriptomic_assays$Kids_First_Participant_ID))
+```
+## How many samples have transcriptomic assays?
+There are `r kids_pbta_samples_rna` patient cases with transcriptomic assays.
+
+## All time points
+```{r}
+transcriptomic_assays %>%
+  group_by(descriptors) %>%
+  tally() %>%
+  arrange(desc(n)) %>% 
+  regulartable() %>%
+  fontsize(size = 12, part = "all")
+```
+
+```{r}
+transcriptomic_assays  %>%
+  group_by(Kids_First_Participant_ID, descriptors, experimental_strategy) %>%
+  tally() %>%
+  arrange(desc(n)) %>% 
+  regulartable() %>%
+  fontsize(size = 12, part = "all")
+```
+
+```{r}
+transcriptomic_assays_matched <- transcriptomic_assays %>%
+  filter(descriptors %in% c("Diagnosis, Recurrence", "Deceased, Diagnosis", "Diagnosis, Progressive",
+                            "Deceased, Recurrence", "Progressive, Recurrence", "Deceased, Progressive", "Deceased, Diagnosis, Progressive")) %>%
+  mutate(experimental_strategy = gsub(";WGS|WXS","", experimental_strategy)) %>%
+  write_tsv(file.path(results_dir, "transcriptomic_assays_matched_time_points.tsv"))
+
+transcriptomic_assays_matched %>%
+  group_by(Kids_First_Participant_ID, descriptors, experimental_strategy) %>%
+  tally() %>%
+  arrange(desc(n)) %>%
+  regulartable() %>%
+  fontsize(size = 12, part = "all")
```

Comment:
Each timepoint (`tumor_descriptor`) has its own RNA-Seq assay. To avoid confusion we decided to show this as a single entry `RNA-Seq` instead of `RNA-Seq; RNA-Seq`. 
You should be seeing one RNA-Seq assay as shown below. Please let me know if this answers your question, or you meant something else.

![Screenshot 2023-08-11 at 8 14 13 PM](https://github.com/d3b-center/pbta-tumor-evolution/assets/46328680/fa7d6e2e-b152-41d5-9ef6-73dd29a3bcfd)